toolchain/check/convert.cpp

```diff
@@ -449,8 +451,8 @@ static auto ConvertStructToStructOrClass(Context& context,
     // Find the matching source field.
     auto src_field_index = i;
     if (src_type.fields_id != dest_type.fields_id) {
-      auto src_field_it = src_field_indexes.find(dest_field.name_id);
-      if (src_field_it == src_field_indexes.end()) {
+      auto src_field_lookup = src_field_indexes.Lookup(dest_field.name_id);
+      if (!src_field_lookup) {
```

Comment:
Optional, but looking at this, I was wondering if it'd be easier to read as:

```suggestion
      if (auto lookup = src_field_indexes.Lookup(dest_field.name_id)) {
        src_field_index = lookup.value();
      } else {
```